npm/dz5/task1.js

```diff
@@ -0,0 +1,17 @@
+const averageGrade = 91;
+
+if (averageGrade <=60) {
```

Comment:
В завданні сказано, що 60 вже склав, тому треба виправити:
```suggestion
if (averageGrade <60) {
```